File: src/main/java/com/back/mozu/domain/setting/repository/HolidayRepository.java
package com.back.mozu.domain.setting.repository;

import com.back.mozu.domain.setting.entity.Holiday;
import java.time.LocalDate;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface HolidayRepository extends JpaRepository<Holiday, LocalDate> {
    List<Holiday> findByDateBetweenOrderByDateAsc(LocalDate startDate, LocalDate endDate);
}

File: src/main/java/com/back/mozu/domain/setting/repository/HolidayRepository.kt
package com.back.mozu.domain.setting.repository

import com.back.mozu.domain.setting.entity.Holiday
import org.springframework.data.jpa.repository.JpaRepository
import java.time.LocalDate

interface HolidayRepository : JpaRepository<Holiday, LocalDate> {
    fun findByDateBetweenOrderByDateAsc(startDate: LocalDate, endDate: LocalDate): List<Holiday>
}